test/tests/classTransformer.test.ts

```diff
@@ -0,0 +1,118 @@
+import { ObjectId } from 'bson';
+import { classToPlain, Exclude, Expose, plainToClass, Transform } from 'class-transformer';
+
+import { prop } from '../../src/prop';
+import { getModelForClass } from '../../src/typegoose';
```

Comment:
```suggestion
import { getModelForClass, prop } from '../../src/typegoose';
```

Reply:
Actually `toStrictEqual` is stricter than `toEqual` : see https://jestjs.io/docs/en/expect#tostrictequalvalue

Reply:
I'm gonna add your changes too.